tfhe/src/js_on_wasm_api/js_high_level_api/integers.rs

```diff
@@ -1022,7 +1022,7 @@ define_expander_get_method!(
 #[wasm_bindgen]
 impl CompactCiphertextListExpander {
     #[wasm_bindgen]
-    pub fn get_bool(&mut self, index: usize) -> Result<FheBool, JsError> {
+    pub fn get_bool(&self, index: usize) -> Result<FheBool, JsError> {
         catch_panic_result(|| {
             self.0
                 .get::<crate::FheBool>(index)
```

Comment:
maybe a better fix is required here as we have an unwrap ?

Reply:
Added a fix